eduvpn/ui/stats.py

```diff
@@ -0,0 +1,138 @@
+import os
+import logging
+import time
+from datetime import timedelta
+
+from ..nm import get_iface, get_ipv4, get_ipv6, get_timestamp
+
+logger = logging.getLogger(__name__)
+
+LINUX_STATS_FILE = "/sys/class/net/{iface}/statistics/{filename}"
```

Comment:
did you try the `Path` from `pathlib`? that is quite useful and a bit more modern than manual path formatting.

Reply:
I didn't think it was really needed here as I'm just formatting the path once, but it does look neater